relay-protocol-derive/src/lib.rs

```diff
@@ -662,6 +662,7 @@ struct FieldAttrs {
 enum SkipSerialization {
     #[default]
     Never,
+    Always,
```

Comment:
Since we want to parse `is_remote` but always serialize it as `is_segment`, I figured we could use an "always skip" mode.